src/app/shared/components/template/components/button/button.component.ts

```diff
@@ -13,6 +13,7 @@ interface IButtonParams {
     | "card-portrait"
     | "flexible"
     | "full"
+    | "google_sign_in"
```

Comment:
nit(blocking): Given that the google_sign_in button has very specific behaviour I think it would be best if not included in default button component as a variant, but instead be it's own standalone component (e.g. `google_sign_in_button` )